### PR TITLE
new compute and stuff

### PR DESCRIPTION
**Pull request by Guria**

### CODE REVIEW

**Comment by Guria** [comment]
now available on @test npm tag

File: packages/cerebral/src/utils.js [hunk 1]
```diff
@@ -142,4 +150,105 @@ export function getProviders (module) {
...
+    value (arg) {
+      if (arg instanceof Tag || arg instanceof Compute) {
+        return arg.getValue(getters)
+      }
+
+      return arg
+    },
+    path (arg) {
+      if (arg instanceof Tag) {
+        return arg.getPath({
```
Comment (Guria) [inline packages/cerebral/src/utils.js]: arg.getPath(getters)

Comment (christianalfoni) [inline packages/cerebral/src/utils.js]: Ah, yeah! Good catch :)

**Comment by Guria** [comment]
smth happened with coverage after rebasing on latest master :(

**Pull request merged**